chromium_src/chrome/browser/ui/commander/tab_command_source.h

```diff
@@ -0,0 +1,30 @@
+// Copyright (c) 2023 The Brave Authors. All rights reserved.
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this file,
+// You can obtain one at https://mozilla.org/MPL/2.0/.
+
+#ifndef BRAVE_CHROMIUM_SRC_CHROME_BROWSER_UI_COMMANDER_TAB_COMMAND_SOURCE_H_
+#define BRAVE_CHROMIUM_SRC_CHROME_BROWSER_UI_COMMANDER_TAB_COMMAND_SOURCE_H_
+
+#include "src/chrome/browser/ui/commander/tab_command_source.h"  // IWYU pragma: export
```

Comment:
I think we can make this subclass to `/brave/browser/ui/commander/` instead of `chromium_src`?
and it seems some others are accidently filtered such as `Close current tab`.

Reply:
Close current tab was deliberately removed, as there is a `Close tab` command already (which is pulled in by BraveSimpleCommandSource).

Similar for `Close other tabs`, `Close tabs to the right`, `Ungroup tab`, `Mute tab`, `Unmute tab`, `Add to Read Later`

Reply:
ah, ok.
also this class reuses upsteam's some methods in the anon namespace! then it should be here :)